cmd/substreams/init.go

```diff
@@ -515,23 +530,61 @@ func runSubstreamsInitE(cmd *cobra.Command, args []string) error {
 			}
 
 		case *pbconvo.SystemOutput_DownloadFiles_:
-			savingDest, _ := os.Getwd()
+			if userState.downloadedFilesfolderPath == "" {
+				savingDest := "output"
+				if projectName := gjson.GetBytes(lastState.State, "name").String(); projectName != "" {
+					savingDest = projectName
+				}
+
+				if cwd, err := os.Getwd(); err == nil {
+					savingDest = filepath.Join(cwd, savingDest)
+				}
```

Comment:
Would have been better to use `filepath.Abs` which does this already and much more.